iotqatools/simulators/listen/python/notification_listener.py

```diff
@@ -1,221 +1,260 @@
-# -*- coding: utf-8 -*-
```

Comment:
As notification_listener.py has been completely changed, the diff is very confusing. I'd recommend to look to the new file directly.